### ISSUE TITLE
Cannot open files if referenced via symlink

### ISSUE THREAD

**Issue by theresajayne** [jonaski/strawberry#340]
If you have your media under a "shortcut" symlink if you try to drag and drop the file onto the player it just reports that it cannot find the file meaning it is not able to follow symlinks

**Expected behavior**
I would expect that the symlink be followed or dereferenced by the program to load the original file in the original location.

Ubuntu 18.4 LTS 
 Strawberry 0.6.5

**Comment by jonaski** [comment]
What filesystem do you use? Where do you symlink the files to?
I have ~/Music symlinked to a drive mouted to /mnt/ and it works fine dragging a file from ~/Music to the playlist here.

**Comment by theresajayne** [comment]
ext4, the folder is /media/resources/music
there is a symlink on ~/Desktop/Music
when you double click on the Link it shows as ~Username/Desktop/Music 
rather than the original location but Strawberry doesn't follow the symlink when dragged from the softlink location (we cannot create hardlinks its restricted on the Desktop Folder)
If the file is dragged from the /media/resources/music folder it works fine.

**Comment by jonaski** [comment]
I've just tested this on Ubuntu 18.04.
I've symlinked /home/jonas/Music to /media/Music
It works just fine dragging a file from Nautilus from both the symlinked directory and the one on /media to the playlist.
The sourcecode that checks if the file can be read is here: https://github.com/jonaski/strawberry/blob/master/src/core/songloader.cpp#L260
It's a normal QFile that follows symlinks.
Either your symlink is broken or you have a permission issue somewhere.
It doesn't work when running as a snap, removable-media needs to be connected first.
`snap connect strawberry:removable-media`
